src/errmsg.f

```diff
@@ -1,23 +1,18 @@
 C> @file
-C
-C> SUBPROGRAM: ERRMSG         WRITE A MESSAGE TO STDERR
-C>   PRGMMR: IREDELL          ORG: W/NMC23     DATE: 95-10-31
+C> @brief Write a message to stderr.
+C> @author Mark Iredell @date 1995-10-31
 C>
-C> ABSTRACT: WRITE A MESSAGE TO STDERR.
+C> Write a message to stderr.
 C>
-C> PROGRAM HISTORY LOG:
-C>   95-10-31  IREDELL
+C> Program history log:
+C>  - 1995-10-31  Mark Iredell
 C>
-C> USAGE:    CALL ERRMSG(CMSG)
-C>   INPUT ARGUMENTS:
-C>     CMSG         CHARACTER*(*) MESSAGE TO WRITE
+C> Usage: call errmsg(cmsg)
```

Comment:
Take out all usage lines. Doxygen provides this automatically based on the actual function declaration.